torch/distributed/nn/api/remote_module.py

```diff
@@ -50,6 +57,16 @@ def __init__(
         The arguments of ``forward_async`` and ``forward`` are the same as
         the ``forward`` method of the module returned by the ``module_cls``.
 
+        Since all the other methods except ``forward`` and ``forward_async``
+        directly inherit from ``nn.Module``,
+        these methods are only applied to the caller on the local side,
+        instead of the user-provided ``module_cls`` that will be executed on the remote node.
+        Therefore, normally these methods should not be used, and the usage will throw exceptions.
+        Particularly, to create a hybrid model, typically the local modules should be
+        in parallel with remote modules, rather than as submodules of any remote module.
+        See the hybrid model example:
+        fbsource/fbcode/caffe2/torch/fb/distributed/model_parallel/docs/README.md
```

Comment:
This is an internal file (anything with fb in the path is internal only), let's not reference internal only code in OSS.